content/en/blog/_posts/2021-03-30-Kubernetes-Multi-Tenancy-Usage-Models.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: blog
+title: 'Two Common Multi-Tenancy Models For Kubernetes'
+date: 2021-03-30
+slug: kubernetes-multi-tenancy-usage-models
+---
+
+**Authors:** Ryan Bezdicek (Medtronic), Jim Bugwadia (Nirmata), Tasha Drew (VMware),  Fei Guo (Alibaba), Adrian Ludwin (Google)
+
+Kubernetes clusters are typically used by several teams in an organization. Secure sharing of Kubernetes resources allows maximizing productivity and saving costs. In other cases, Kubernetes may be used to deliver software and platform services to end users requiring segmentation and isolation of resources across users.
+
+The Kubernetes Multi-Tenancy Working Group is chartered with defining multi-tenancy models for Kubernetes and making it easier to operationalize multi-tenant use cases.  This blog post, from the multi-tenancy working group members, describes two common usage models for multi-tenancy and introduces working group projects that can be used to simplify Kubernetes multi-tenancy management. 
+
+We will also be presenting on this content and discussing different use cases at our Kubecon EU 2021 panel session, [Multi-tenancy vs. Multi-cluster: When Should you Use What?](https://sched.co/iE66).
+
+## Multi-tenancy security considerations
+
+Cloud native security involves different system layers and lifecycle phases as described in the [Cloud Native Security Whitepaper](https://www.cncf.io/blog/2020/11/18/announcing-the-cloud-native-security-white-paper/) from CNCF SIG Security.  Without proper security measures implemented across all layers and phases, Kubernetes multi-tenancy can be compromised. 
```

Comment:
"K8s multi-tenancy can be compormised"

Maybe we can be a bit more specific? "..., a problem with one tenant can threaten all other tenants on the same cluster."